.circleci/config.yml

```diff
@@ -0,0 +1,85 @@
+version: 2
+jobs:
+  lint:
+    docker:
+      - image: circleci/golang:1.14
+    steps:
+      - checkout
+      - run: make lint
+
+  test:
+    docker:
+      - image: circleci/golang:1.14
+    steps:
+      - checkout
+      - run: make test-and-coverage
+
+  build:
+    docker:
+      - image: circleci/golang:1.14
+    steps:
+      - checkout
+      - run: make build SLO_EXPORTER_VERSION=${CIRCLE_TAG:-$CIRCLE_SHA1}
+      - persist_to_workspace:
+          root: .
+          paths:
+            - build
+
+  docker_release:
+    docker:
+      - image: circleci/golang:1.14
+    environment:
+      DOCKERHUB_USERNAME=sklikdevops
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 17.07.0-ce
+      - attach_workspace:
+          at: .
```

Comment:
Not familiar with this, is it necessary?

Reply:
I believe it is. In documentation it is referred to as a mandatory option for [`attach_workspace`](https://circleci.com/docs/2.0/configuration-reference/#attach_workspace)